posts/rise-of-microservices-and-docker.md

```diff
@@ -0,0 +1,93 @@
+# Rise of Microservices and Docker
+
+Are you a software engineer/developer/coder or maybe even a tech enthusiast who hears terms like `Microservices` and `Docker`? 
+
+Are you someone who has been encountering these terminologies so often through various channels like articles, news, blogs, meetups, conferences but still clueless there's so much hype?
```

Comment:
I don't think this is necessary.

Reply:
Hey Sid, I have made the required changes wrt to the images. I have used LucidChart as suggested by you and converted all hand drawn images to digital ones.

Also, I feel that is good start because it helps the reader build some context. I want them to recall all the possible occasions and scenarios where they have come across these words but nevertheless what do you think could be a good start?